README.md

```diff
@@ -1,40 +1,45 @@
-This repo can be ran on its own or as part of
-[Optimism Integration](https://github.com/ethereum-optimism/optimism-integration).
+## Structure
+Each test suite is defined as its own package inside the [./packages](./packages) directory.
+This structure simplifies the process of executing test suites selectively.
+Packages must not depend on one other.
 
+## Running via optimism-integration
+[optimism-integration](https://github.com/ethereum-optimism/optimism-integration) can be used to automatically configure and run the test suites found in this repository.
+Docker images found on [Dockerhub](https://hub.docker.com/u/ethereumoptimism) are used when running the test suites this way.
+Refer to the [README](https://github.com/ethereum-optimism/optimism-integration/blob/master/README.md) for detailed usage information.
 
-## Running with Optimism Integration
+## Running locally (aka how to write new tests)
+Tests in this repository can be executed against a set of local services, but all configuration has to be handled by the user (aka you).
+You'll probably need to do this if you plan to develop in this repo (e.g., to write new tests).
 
-The [Optimism Integration](https://github.com/ethereum-optimism/optimism-integration)
-repo can be used to automatically configure and run the test suites found
-in this repo. See the [README](https://github.com/ethereum-optimism/optimism-integration/blob/master/README.md)
-for usage information.
+First, set up a `.env` file:
+1. Make a copy of `.env.example` and name it `.env`.
+2. Set the variables inside the `.env` file to your desired variables (default will likely suffice).
 
-It is assumed that each test suite gets a fresh state when running this way.
-This means that the packages must not depend on each other.
+Now start up the services that you want to test.
+The easiest way to run every Optimistic Ethereum service is to use the [up.sh](https://github.com/ethereum-optimism/optimism-integration/#upsh) script.
+Follow the instructions in [optimism-integration](https://github.com/ethereum-optimism/optimism-integration#optimism-integration) to get this set up.
 
-Docker images found on [Dockerhub](https://hub.docker.com/u/ethereumoptimism)
-are used when running the test suites this way.
-
-## Running locally
-
-This repo can be ran locally against existing services but the configuration
-must be handled by the user. A `.env` file will be used to populate the
-configuration when then environment variable `NODE_ENV` is set to `local`.
-
-There should be a `yarn` script for each package found in `packages`.
+Finally, run the tests.
+At the moment you'll need to set environment variable `NODE_ENV=local` to signal that you want to run these tests locally:
+```bash
+NODE_ENV=local yarn test
+```
 
+You can also run specific tests via:
 ```bash
-$ yarn test:x-domain
-$ yarn test:tx-ingestion
+NODE_ENV=local yarn test:x-domain
+NODE_ENV=local yarn test:tx-ingestion
```

Comment:
Are you sure we need `NODE_ENV` here?

Reply:
I think so? https://github.com/ethereum-optimism/integration-tests/blob/d806129050e920c0a10436b4c8fce5141cb1bf42/common.ts#L10-L14

I had to set this variable manually for that line to trigger.

Reply:
See https://github.com/ethereum-optimism/integration-tests/blob/3169ebc50728f6825ade23d106201d55c16810ab/package.json#L16